### PR TITLE
PACT-720 Add and configure subscribe channel to the AsyncAPI schema

### PR DESCRIPTION
**Pull request by felixgb**

### CODE REVIEW

File: schema/omega-services.async.api.yaml [hunk 1]
```diff
@@ -48,6 +48,17 @@ channels:
         oneOf:
           - $ref: '#/components/messages/getEditSet'
 
+  omega.editorial.web.application.instance.1:
+    description: The queue on which response messages will be sent.
+    publish:
```
Comment (rwalpole) [inline schema/omega-services.async.api.yaml:53]: I think this should be `subscribe:`, as in this is the channel that client's will subscribe to receive their responses. They won't publish here.

File: schema/omega-services.async.api.yaml [hunk 2]
```diff
@@ -103,6 +114,16 @@ components:
         description: Default Correlation ID
         location: $message.header#/correlationId
 
+    editSetRecord:
+      name: editSetSummary
```
Comment (rwalpole) [inline schema/omega-services.async.api.yaml:118]: This should be `editSetRecord`

File: schema/omega-services.async.api.yaml [hunk 3]
```diff
@@ -103,6 +114,16 @@ components:
...
+      title: EditSet Summary
```
Comment (rwalpole) [inline schema/omega-services.async.api.yaml:119]: This should be EditSet Record

File: schema/omega-services.async.api.yaml [hunk 4]
```diff
@@ -48,6 +48,17 @@ channels:
...
+  omega.editorial.web.application.instance.1:
+    description: The queue on which response messages will be sent.
+    publish:
+      operationId: omega_editorial_web_application_instance_1
+      description: Response messages
+      message:
+        oneOf:
+          - $ref: '#/components/messages/editSet'
+          - $ref: '#/components/messages/editSetSummary'
+          - $ref: '#/components/messages/editSetRecord'
```
Comment (rwalpole) [inline schema/omega-services.async.api.yaml:60]: I think we want `updateEditSetRecord` here as well. Basically anything that can appear in this queue.

**Review by rwalpole — CHANGES_REQUESTED** [review]

File: schema/omega-services.async.api.yaml (hunk 4 above)
Comment (felixgb) [reply schema/omega-services.async.api.yaml:60]: I've added the update message here, as well as the update message confirmation

**Review by rwalpole — APPROVED** [review]

**Pull request merged**